cfgov/v1/atomic_elements/organisms.py

```diff
@@ -570,13 +570,17 @@ class Meta:
 
 class BaseExpandable(blocks.StructBlock):
     label = blocks.CharBlock(required=False)
+    icon = blocks.CharBlock(required=False)
     is_bordered = blocks.BooleanBlock(required=False)
     is_midtone = blocks.BooleanBlock(required=False)
     is_expanded = blocks.BooleanBlock(required=False)
+    larger_heading = blocks.BooleanBlock(required=False)
+    expanded_padding = blocks.BooleanBlock(required=False)
```

Comment:
I would probably follow the `is_` style of the other boolean values above here. There's a boolean naming convention to name the variable name like a question with either a prefixed `is_` or a `has_`, which can be answered with "true" or "false" (see examples in https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig).

Reply:
changed in https://github.com/cfpb/consumerfinance.gov/pull/8191/commits/374073de5799829c008bc8e4281eddfd4d57220a